geoAssembler/qt/app.py

```diff
@@ -110,6 +109,11 @@ def __init__(self, app, run_dir=None, geofile=None, levels=None):
         if run_dir:
             self.run_selector.read_rundir(run_dir)
 
+    def close(self):
```

Comment:
I believe it's better to use `def closeEvent(event)` here, since it catches all forms of closing (some of which are not triggered by `close()`)

Reply:
I was overriding this because the tests explicitly call `.close()`, e.g. here:

https://github.com/European-XFEL/geoAssembler/blob/ca5737eecd5f32b924ce6ddaf7a03037f11086e1/geoAssembler/tests/conftest.py#L70-L72

Does a call like that also trigger `.closeEvent()`? Or is there some signal that we should connect to?

I think we've only seen this so far in the tests, which create and tear down the GUI several times. It's not clear to me if there's any problem with the actual application, so I focused on making the tests work. :-)

Reply:
Yes, that is triggered as well. Overriding Qt events are more preferred as it compasses wider range of situations (e.g., widget flags, other events). 

I am alright with the current implementation :) For this main window, the difference shouldn't matter much

Reply:
Thanks Cammille - I've changed it to `closeEvent`, and the tests still pass on my computer.